testing/features/components/step_definitions/common_steps.rb

```diff
@@ -55,3 +55,7 @@
 
   expect(@component).not_to have_search_field
 end
+
+Then("a/an {string} label is present above the title") do |label|
+  expect(@component.label.text).to eq(I18n.t("cads.callout.#{label.downcase}"))
```

Comment:
I know we've circled round this in the past but could we avoid using the internal I18n keys in the tests. Looks like `expect(@component.label.text).to eq(label))` might have a comparable effect where we pass the expected label directly?

Reply:
For some of them (such as here), we can do that. But for some we need to keep it in. Up to you what we do for consistency here.

Reply:
Let's take it out please. We probably should catch up about language tests specifically in the design system because the new engine / view components change the boundaries of that a little bit anyway.